_includes/head.html

```diff
@@ -18,4 +18,8 @@
     <!-- Custom styles for this template -->
     <link href="css/grayscale.css" rel="stylesheet">
 
+
+    <!-- Custom styles for this slider -->
+    <link href="css/full-slider.css" rel="stylesheet">
```

Comment:
you can put the things that are in this css into the grayscale.css file so that you will have only one file to care about :+1: